ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -495,6 +495,8 @@ extension DispatchTimeInterval {
 			return TimeInterval(UInt64(us) * NSEC_PER_USEC) / TimeInterval(NSEC_PER_SEC)
 		case let .nanoseconds(ns):
 			return TimeInterval(ns) / TimeInterval(NSEC_PER_SEC)
+		case let .never:
```

Comment:
This needs to be conditionally compiled (just like we did in ReactiveSwift) to maintain compatibility with Xcode 8.2/Swift 3.1.